app-rating/src/main/java/com/stepstone/apprating/ratingbar/CustomRatingBar.kt

```diff
@@ -104,6 +112,17 @@ class CustomRatingBar(context: Context, attrs: AttributeSet) : LinearLayout(cont
         this.onRatingBarChangedListener = onRatingBarChangedListener
     }
 
+    private fun getStarColor(context: Context): Int {
+        if (starColorResId != 0) {
+            return if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Same as above, you could use `ResourcesCompat.getColor()` instead

Reply:
corrected